main.tf

```diff
@@ -1,3 +1,23 @@
-resource "random_pet" "template" {
-  length = var.length
+locals {
+  dd_team = {
+    for member in var.users :
+    member.username => member if member.access_roles["datadog"]
+  }
+}
+
+data "datadog_role" "standard" {
+  filter = "Datadog Standard Role"
+}
+
+resource "datadog_user" "users" {
+  for_each             = local.dd_team
+  disabled             = each.value.disabled
+  email                = each.value.email
+  name                 = each.value.name
+  roles                = [data.datadog_role.standard.id]
+  send_user_invitation = each.value.send_user_invitation
+
+  lifecycle {
+    ignore_changes = [name]
```

Comment:
Let's include a comment on why we lifecycle ignore `name`. I think it's because it gets sets to the users email or something until they first sign in?

Reply:
I don't see any naming modifications in our Clients' Datadog. Also, couldn't find any explanation why it was added initially. 
I removed this lifecycle, because we can't explain why this is here.